thinkhazard/static/less/report.less

```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
```

Comment:
I wouldn't set a height here. Even if specified in the document. The size should automatically be set given the element content by the browser.